examples/for-tests/src/App.js

```diff
@@ -746,6 +746,14 @@ function getPasswordlessConfigs({ disableDefaultImplementation }) {
             disableDefaultImplementation,
             style: theme.style,
         },
+        translations: {
```

Comment:
We should move `translations` to supertokens.init and not to have a recipe level, and club all the translation related config into one object as opposed to keeping them all in the root level. You can call this `translationsForLanguages` or `languageTranslation`. The fields in the object are:
- `translations` (contains the variable key values for different diff langs
- `t` function override
- `defaultLanguage`
- `translationScope`